h/static/scripts/group-forms/components/forms/Checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import type { CheckboxProps as BaseCheckboxProps } from '@hypothesis/frontend-shared';
+import { Checkbox as BaseCheckbox } from '@hypothesis/frontend-shared';
+import type { ComponentChildren } from 'preact';
+import { useId } from 'preact/hooks';
+
+export type CheckboxProps = Omit<BaseCheckboxProps, 'aria-describedby'> & {
+  /**
+   * Adds a description right under the checkbox and main content, aligned with
+   * the main content's left side.
+   */
+  description?: ComponentChildren;
+
+  'aria-describedby'?: string;
+};
+
+export default function Checkbox({
+  description,
+  children,
+  'aria-describedby': ariaDescribedBy,
+  ...checkboxProps
+}: CheckboxProps) {
+  const descriptionId = useId();
+
+  const describedBy: string[] = [];
+  if (ariaDescribedBy) {
+    describedBy.unshift(ariaDescribedBy);
+  }
```

Comment:
Is there a planned use for passing `ariaDescribedBy` instead of `description`? It seems to me that if you're not using `description`, you could just use the base `Checkbox`.

Reply:
Yeah, I think that makes sense. Returning the BaseCheckbox as is when no description is provided, and only adding the description otherwise.

Supporting both a description and another element also describing the checkbox is probably an edge case and usually overkill.